packages/common/tslint.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file may be redundant (from a code editor's perspective). Only the root one might be enough?